gosuper/device/device.go

```diff
@@ -7,10 +7,10 @@ import (
 	"sync"
 	"time"
 
-	"resin-supervisor/gosuper/config"
-	"resin-supervisor/gosuper/resin"
-	"resin-supervisor/gosuper/supermodels"
-	"resin-supervisor/gosuper/utils"
+	"github.com/resin-io/resin-supervisor/gosuper/config"
```

Comment:
@pcarranzav Ah okay, but why do you do the rewriting? I would also strongly recommend using the go native vendoring: https://golang.org/s/go15vendor over Godeps (which is IMO a PITA) ;)